ibm_mq/tox.ini

```diff
@@ -16,5 +16,6 @@ commands =
     pip install -r requirements.in
     pytest -v
 setenv =
+    LD_LIBRARY_PATH=/opt/mqm/lib64:/opt/mqm/lib:{env:LD_LIBRARY_PATH:none}
```

Comment:
Are we sure this path is generalized to *nix and not Travis-runner specific?

Reply:
Tested on Ubuntu and Centos, that's the default location where those libs are installed.

Reply:
Cool thanks! And macOS?

Reply:
Seems the installation is manual on Mac OS, so no default location I think.

Julia made this PR a while ago: https://github.com/DataDog/integrations-core/pull/3895/files?short_path=da6c50e#diff-da6c50e20af53462d382032935fe10b7

But having LD_LIBRARY_PATH in the `tox.ini` should not impact negatively since user can still declare more LD_LIBRARY_PATH that will be appended.